src/index.js

```diff
@@ -12,7 +12,7 @@ const displayScores = (list, res) => {
   }
   list.innerHTML = '';
   scores.forEach((score) => {
-    list.innerHTML += `<li>${score[0]}: ${score[1]}</li>`;
+    list.innerHTML += `<li class="users"><div class="user-img"><img src="../img/mario.png" alt="User Avatar"></div><span class="score-name">${score[0]}</span><span>${score[1]}</span></li>`;
```

Comment:
### [Optional]

It would be nice to fix the avatar, it is broken.

Perhaps try `../img/mario.png`